hooks/tweet.py

```diff
@@ -17,8 +17,11 @@
 
 CONSUMER_KEY    = "9Hb6ZdMmvuAWxi4GCkfhToRiH"
 CONSUMER_SECRET = "86kx9Mv9wJ5UTkDEw2jRBFYstpkDK2iP7ZAo12fhf0WooMln5w"
-
-import twitter
+try:
+    import twitter
+except ModuleNotFoundError:
+    print("tweet-hook: python3-twitter is not installed.")
+    return 1
```

Comment:
This looks OK, but it should be in another pull request, because this was about Kitsu's mediatypes.

Reply:
How can I add a commit in a different PR?

Reply:
Is this a return at module level? (Amon mobile.)

Reply:
Fixed in 41c6b4b.